setup.py

```diff
@@ -1,26 +1,18 @@
 #!/usr/bin/env python
 
+import glob
 import sys
 import os
 import warnings
 
+from setuptools import setup, Extension
 from distutils.dist import Distribution
 
 display_option_names = Distribution.display_option_names + ['help', 'help-commands']
 query_only = any('--' + opt in sys.argv for opt in display_option_names) or len(sys.argv) < 2 or sys.argv[1] == 'egg_info'
 
-# Use setuptools for querying the package, normal builds use distutils
```

Comment:
@mrjbq7 The requirement is quite strict. I've left it unchanged. The problem I met should only be happened in my environment and there might be some problem with my virtualenv.